doc-content/enterprise-only/admin-and-config/dm-server-configure-proc.adoc

```diff
@@ -1,17 +1,17 @@
 [id='dm-server-configure-proc']
 = Configuring the {PRODUCT} server
 
-You can configure Red Hat JBoss Developer Studio to run the {PRODUCT} server.
+You can configure Red Hat Developer Studio to run the {PRODUCT} server.
 
 .Prerequisites
-* Red Hat JBoss Developer Studio is installed with the {PRODUCT} plug-ins.
+* Red Hat Developer Studio is installed with the {PRODUCT} plug-ins.
 * {Product} is installed in a {EAP} 7.1 server instance.
```

Comment:
We are only support EAP 7.2 with this release -  however, I am hesitant to make this change without you confirming with QE.

Reply:
OK, confirmed with Karel Suta. Should be {EAP} 7.2. Search the docs to be sure no other instances.

Reply:
Will apply this change to the 7.2 branch.